tools/common/pathutil.flow

```diff
@@ -130,3 +142,23 @@ fcCorrectUriPath(path : string) -> string {
 		}
 	}
 }
+
+copyFileRelative(base : string, from : string, to : string) -> void {
+	if (!fileExists(from)) {
+		print("file: '" + from + "' doesn't exist - not possible to copy");
+	} else {
+		name = strRight(from, strlen(base));
+		path_to = pathCombine(to, name);
+		ensureDirectoryExists(dirName(path_to));
+		if (!copyFile(from, path_to)) {
+			println("error while copying file '" + path_to + "' from '" + from + "' to '" + to + "'");
```

Comment:
Why not change this to return a string with an error, so the caller can detect if it was done ok.

Reply:
Copy functions have changed so that they return an error message / array of error messages